### PR TITLE
EPMEDU-840: Remove bottomnav labels

### PR DESCRIPTION
**Pull request by carlossantiagomenendez**
Deletion of bottomnavbar labels

### CODE REVIEW

File: feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt [hunk 1]
```diff
@@ -97,9 +97,6 @@ private fun BottomNavigationBar(
                             contentDescription = stringResource(item.title)
                         )
                     },
-                    label = {
-                        Text(text = stringResource(id = item.title))
```
Comment (egorikftp) [inline feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt]: What about `title` field inside `NavigationTab`? Is it use now?

Comment (carlossantiagomenendez) [reply feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt]: It no longer has any use.

**Review by Fileek — APPROVED** [review]

File: feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt (hunk 1 above)
Comment (egorikftp) [reply feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt]: please rename `title` to `contentDescription`

Comment (carlossantiagomenendez) [reply feature/tabs/src/main/java/com/epmedu/animeal/tabs/presentation/TabsScreen.kt:101]: Changes done

**Comment by carlossantiagomenendez** [comment]
Changes applied

**Review by n3gbx — APPROVED** [review]

**Pull request merged**